src/config.rs

```diff
@@ -44,7 +41,14 @@ cfg_if::cfg_if! {
     }
 }
 pub const MAX_SHORT_DATA_LENGTH: usize = 128;
-pub const MAX_SIGNATURE_LENGTH: usize = 72;
+
+pub const MAX_SIGNATURE_LENGTH: usize = 512 * 2;
+// FIXME: Value from https://stackoverflow.com/questions/5403808/private-key-length-bytes for Rsa2048 Private key
+pub const MAX_KEY_MATERIAL_LENGTH: usize = 1160 * 2 + 72;
```

Comment:
Could this be features flagged behind a `big-keys` feature perhaps? Although I'm not sure if the request enum is "big" anyway due to other variants. The thinking is that for non-RSA builds, keys are more like 32 bytes than 2-3KB.

Reply:
The total increase to the interchange size is only 100 bytes. I'll put it behind a feature-flag.

Reply:
Yeah I agree 100 bytes is not worth the effort.